dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsApiExtendedServiceImpl.java

```diff
@@ -689,6 +714,35 @@ private Optional<? extends Version<?>> getVersion(Entry<?, ?> entry, String vers
         return versionOptional;
     }
 
+    /**
+     * Get candidate version that can be considered for an ai Topic (or other?)
+     * @param entry
+     * @return
+     */
+    private Optional<? extends Version<?>> getVersionCandidate(Entry<?, ?> entry) {
+        if (entry instanceof Workflow workflow) {
+            List<WorkflowVersion> workflowVersions = workflowVersionDAO.getWorkflowVersionsByWorkflowId(workflow.getId(), 1, 1);
```

Comment:
Should the last `1` be `0` since the first result is index 0?

Reply:
Good point, 
off-by-1 you old friend